### ISSUE TITLE
Does latex-bundle work oth Symfony 7.1.*?

### ISSUE THREAD

**Issue by mario6097** [bobvandevijver/latex-bundle#51]
Hi
does latex-bundle work oth Symfony 7.1.*?
If I try to install with a usual composer updat, I get
` bobv/latex-bundle v5.0.2 requires symfony/config ^4.4 || ^5.4 || ^6.2 -> found symfony/config[v4.4.0, ..., v4.4.44, v5.4.0, ..., v5.4.40, v6.2.0, ..., v6.4.8] but it conflicts with your root composer.json require (7.1.*).`

Is there a way to fix?
Thanks

**Comment by bobvandevijver** [comment]
I haven't tested it yet with Symfony 7 as we normally stay on LTS. Can you please try whether `dev-sf7` works as expected for you?

**Comment by mario6097** [comment]
Yes, it seems it works. (I had tried without success, maybe I had a typo). 
Which LTS version do you use? 
Thanks so much

**Comment by bobvandevijver** [comment]
Current Symfony LTS is 6.4, but great to hear that the update works. I will be merging and tagging it.
